tensorboard/components/tf_card_heading/tf-card-heading.html

```diff
@@ -62,25 +82,25 @@
         </paper-dialog-scrollable>
       </paper-dialog>
     </div>
-    <style>
+    <style include="tf-card-heading-style">
       .container {
-        border-left: 4px solid;  /* border-color set as inline style */
-        padding-left: 5px;
-        margin-bottom: 10px;
         display: flex;
       }
       .content {
         font-size: 12px;
         flex-grow: 1;
       }
-      .label {
-        display: block;
-        text-overflow: ellipsis;
-        overflow: hidden;
-      }
-      .label:first-child {
+      .name {
         font-size: 14px;
       }
+      .run {
+        font-size: 11px;
+        width: auto;
+        border-radius: 3px;
+        text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
```

Comment:
FYI, I personally avoid text-shadow due to performance issues.
https://stackoverflow.com/questions/32948596/is-csss-efficiency-with-text-shadow-as-bad-as-box-shadow

but this is a small text shadow, and our frontend is not bottlenecked by browser paints, so I think we're good.

Reply:
Noted.